### PR TITLE
update MX

### PR DESCRIPTION
**Pull request by sherryw94**

### CODE REVIEW

File: public/data/travel/MX.yml [hunk 1]
```diff
@@ -39,26 +39,25 @@ travel:
   inbound_allowed:
   outbound: partial
   inbound_restricted:
-    - US
```
Comment (jannwong) [inline public/data/travel/MX.yml:42]: why is this removed?

**Review by jannwong — CHANGES_REQUESTED** [review]
inline comment

File: public/data/travel/MX.yml (hunk 1 above)
Comment (sherryw94) [reply public/data/travel/MX.yml:42]: hello jann! i have clarified with gab about info to include, so we will only be including info for tourists when it comes to inbound travel/outbound/restricted. i initially thought to remove this because they allow essential travel to re-enter via borders. will rectify

**Review by jannwong — APPROVED** [review]
